mobile.css

```diff
@@ -0,0 +1,110 @@
+.body {
+    max-width: 375px;
+    margin: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)